_monitoring-plugins/alerting/api.md

```diff
@@ -1438,6 +1509,8 @@ POST _plugins/_alerting/destinations/email_accounts
 Introduced 1.0
 {: .label .label-purple }
 
+When you update an email account, you can use the `seq_no` and `primary_term` to update an existing email account. If these numbers don’t match the existing email account or the email account doesn’t exist, alerting throws an error. OpenSearch increments the version number and the sequence number automatically (see the sample response).
```

Comment:
"When updating a destination, you can include `seq_no` and `primary_term` as part of your request. If these numbers don't match the existing monitor or the monitor doesn't exist, the alerting plugin throws an error. OpenSearch increments the version number and the sequence number automatically (see the sample response)."

Reply:
Thanks for the review, I fixed the description as you mentioned.